s3-uploads.php

```diff
@@ -32,6 +36,34 @@ function s3_uploads_init() {
 }
 
 /**
+ * Check whether the environment meets the plugin's requirements, like the minimum PHP version.
+ *
+ * @return bool True if the requirements are met, else false.
+ */
+function s3_uploads_check_requirements() {
+	if ( version_compare( '5.3.3', PHP_VERSION, '>=' ) ) {
```

Comment:
From what I can gather, this is going to be true if `PHP_VERSION` is `5.3.3` which we don't want. It should be using  `>`.